lib/elastic_apm/error_builder.rb

```diff
@@ -59,10 +50,20 @@ def add_stacktrace(error, kind, backtrace)
       error.culprit = stacktrace.frames.first.function
     end
 
-    def add_current_transaction_fields(error)
-      return unless (transaction = ElasticAPM.current_transaction)
+    # rubocop:disable Metrics/AbcSize
+    def add_current_transaction_fields(error, transaction)
+      return unless transaction
+
       error.transaction_id = transaction.id
       error.transaction = { sampled: transaction.sampled? }
+      error.trace_id = transaction.trace_id
+      error.parent_id = ElasticAPM.current_span&.id || transaction.id
+
+      return unless transaction.context
+
+      Util.reverse_merge!(error.context.tags, transaction.context.tags)
+      Util.reverse_merge!(error.context.custom, transaction.context.custom)
```

Comment:
Why are you merging `tags` and `custom`, is this aligned with how other agents behave? I'd expect to find only the configured tags and customs if specifically set for an error.

Reply:
Some do but it differs so I started https://github.com/elastic/apm/issues/62